### ISSUE TITLE
Handling platform back button

### ISSUE THREAD

**Issue by dannylin108** [YowFung/intro#2]
Platform back button is not handled property. 

If press hardware back button during introduction, it just pops the route, not interrupting the introduction.

Ideally it should optionally act as `controller.previous()` or as `controller.close()`

**Comment by YowFung** [comment]
Hi @dannylin108 
In my opinion, this should be the part you need to implement in your business code.

**Comment by dannylin108** [comment]
Yes.. But if you could make it as a parameter for `Intro` or `IntroStepTarget`, it would be awesome :+1:
